.github/workflows/validate_yaml.yaml

```diff
@@ -1,20 +0,0 @@
-# yamllint disable rule:line-length
```

Comment:
Yes, I'd prefer we keep it because it clean ups the output of yamllint from unnecessary warnings if we ever need to validate this file.

Reply:
Yes, I'd prefer we keep it because if these files you are editing (`ci.yaml`) contain an error, the GitHub actions simply wouldn't run them making an impression everything is good and proper.

@sirreal Unless there's a compelling evidence, can we have these back please?

Reply:
Sure. I'll restore this workflow.